app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
hey @klappradla, we are wondering that maybe this attendance controller it is not being used. We create/update attendances through the nested field in edit team action. Unless you need this data for some API...